ci/cpu/build.sh

```diff
@@ -64,7 +64,7 @@ if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
   CONDA_CHANNEL=""
 else
   CONDA_BUILD_ARGS="--dirty --no-remove-work-dir"
-  CONDA_CHANNEL="-c $WORKSPACE/ci/artifacts/cudf/cpu/.conda-bld/"
+  CONDA_CHANNEL="-c "$WORKSPACE/ci/artifacts/cudf/cpu/.conda-bld/"
```

Comment:
```suggestion
  CONDA_CHANNEL="-c $WORKSPACE/ci/artifacts/cudf/cpu/.conda-bld/"
```

I think this may be fine as it was.